crates/mast-package/src/package/manifest.rs

```diff
@@ -74,19 +87,119 @@ impl PackageManifest {
         digest: &Word,
     ) -> impl Iterator<Item = &PackageExport> + '_ {
```

Comment:
Similar to the above comment - should this be `get_proc_export_by_digest()`? And I'd probably update the comment to say "all procedure exports".

Also, since this is only for procedures, the return type could probably be `impl Iterator<Item = &ProcedureExport>`.

Reply:
I'd change it to `get_procedures_by_digest`, but agreed